tests/pallet-upload-handler-spec.js

```diff
@@ -3,13 +3,19 @@ ngDescribe({
   modules: 'platanus.palletBundle',
   element:
     '<pallet-upload-handler ' +
+      'button-label="Seleccionar" ' +
       'no-document-text="no file present"' +
       'upload-url="uploads"' +
       'progress-type="bar"' +
       'ng-model="user.uploadIdentifier">' +
     '</pallet-upload-handler>',
 
   tests: function(deps) {
+    it('shows custom button label', function() {
```

Comment:
'it' is not defined                                              no-undef<br>Unexpected function expression                                   prefer-arrow-callback<br>Missing space before function parentheses                        space-before-function-paren